packages/dkml-component-staging-ocamlrun/dkml-component-staging-ocamlrun.4.12.1~v1.0.2/opam

```diff
@@ -14,9 +14,15 @@ depends: [
 ]
 dev-repo: "git+https://github.com/diskuv/dkml-component-ocamlrun.git"
 available: [ os = "macos" | (os = "linux" & (arch = "x86_32" | arch = "x86_64")) | os = "win32" ]
```

Comment:
Not a blocker for this PR, but you may want to refine with `os-distribution = "homebrew"`